src/components/Card.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import CardModal from '../components/CardModal'
+import '../styles/Card.css'
+
+type CardProps = {
+  alt?: string
+  media_type?: string
+  title?: string
+  date?: string
+  src: string
+  highDefSrc?: string
+  explanation?: string
+  copyright: string
+  likedImage: boolean
+  innerRef?: (node: HTMLDivElement) => void
+  likeAction?: () => void
+}
+
+function Card({
+  alt,
```

Comment:
Because `title` and `alt` are optional. They have the possibility of rendering "undefined" on your UI if no image or alt value exists. You can assign a default value to a prop and ensure your ternary below will always render a value. Even if it's am empty string. When the `alt` is passed, it will override the default. 
```suggestion
  alt = "",
```